decisions/wc_society_decisions.txt

```diff
@@ -543,4 +543,61 @@ society_decisions = {
 			# factor = 0
 		# }
 	# }
+	forge_hammer_of_the_twilight = {
+		is_high_prio = yes
+
+		ai_check_interval = 24
+
+		only_playable = yes
+		is_in_society = yes
+
+		potential = {
+			is_untouchable_trigger = no
+
+			is_adult = yes
+			is_incapable = no
+			prisoner = no
+
+			society_member_of = twilights_hammer_cult
+			society_rank == 4
+			NOT = { has_global_flag = hammer_of_the_twilight_forged }
+		}
+
+		allow = {
+			is_inaccessible_trigger = no
+
+			religion = old_gods_worship
+			true_religion = old_gods_worship
+
+			society_currency >= 5000
+			OR = {
+				AND = {
+					has_character_modifier = follower_of_nzoth
+					offmap_nzoth = { has_policy = old_god_working }
+				}
+				AND = {
+					has_character_modifier = follower_of_cthun
+					offmap_cthun = { has_policy = old_god_working }
+				}
+				AND = {
+					has_character_modifier = follower_of_yogg_saron
+					offmap_yoggsaron = { has_policy = old_god_working }
+				}
+			}
+		}
+
+		effect = {
+			add_artifact = hammer_of_the_twilight
+			set_global_flag = hammer_of_the_twilight_forged
```

Comment:
Not fixed, still can get 2

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46576860/66940300-64744400-f04d-11e9-889f-f6293cde3b1a.png)
![image](https://user-images.githubusercontent.com/46576860/66940318-6e964280-f04d-11e9-9b35-3109e60ecaf4.png)

</details>

Reply:
not craftable anymore